css/style.css

```diff
@@ -1,18 +1,522 @@
+}
+
+.footer-logo-text{
+  font-size: 22px;
+  line-height: 30px;
+}
+
+.footer-top-block{
+  display: inline;
+}
+
+.copy{
+  font-family: "Poppins", "Roboto", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  line-height: 22px;
+  color: #C2C3C5;
+  font-size: 14px;
+  text-transform: capitalize;
+}
+
+.footer-nav-link{
+  font-family: "Poppins", "Roboto", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  line-height: 22px;
+  color: #C2C3C5;
+  font-size: 14px;
+  text-transform: capitalize;
+}
+
+.footer-nav-item{
+  display: inline;
+}
+
+
+/* CATALOG PAGE */
+
+.page-catalog{
+  background-color: #000;
+}
+
+.welcome-container{
+  background-color: #141B22;
+}
+
+.welcome-container-heading{
+  font-size: 44px;
+  line-height: 56px;
+}
+
+.breadcrumb-link{
+  font-size: 14px;
+  line-height: 20px;
+  color: #888B8E;
+}
+
+.catalog-heading{
+  text-transform: capitalize;
+}
+
+.filter-container {
+  max-width: 300px;
+  margin: 20px auto;
+  background-color: #2b2b2b;
+  padding: 10px;
+}
+
+.toggle-filter {
+  display: none;
+}
+
+.filter-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  cursor: pointer;
+  padding: 10px;
+  background-color: #141B22;
+}
+
+.filter-title {
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.filter-header::after{
+  content: "-";
+}
+
+.toggle-filter:checked + .filter-header::after {
+content: "+";
+}
+
+.toggle-filter:checked ~ .filter-options {
+  display: block;
+}
+
+.toggle-filter ~ .filter-options {
+  display: none;
+}
+
+.filter-options {
+  font-family: "Poppins", "Roboto", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  line-height: 22px;
+  color: #C2C3C5;
+  font-size: 14px;
+  background-color: #141B22;
+}
+
+
+.filter-options label {
+  display: block;
+  padding: 5px 0;
+}
+
+.breadcrumb li::after {
+  content: "|";
+  margin-left: 10px;
+}
+
+.breadcrumb li:last-child::after {
+  content: "";
+}
+
+.breadcrumb li:last-child a {
+  color: #21E786; 
+}
+.search-form{
+background-color: #141B22;
+}
+
+::placeholder{
+  font-family: "Poppins", "Roboto", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  line-height: 26px;
+  color: #888B8E;
+  font-size: 14px;
+  background-color: #141B22;
+}
+
+.button-search-reset{
+  line-height: 22px;
+  color: #21E786;
+  font-size: 14px;
+}
+
+.footer-text{
+  font-family: "Poppins", "Roboto", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  line-height: 26px;
+  color: #fff;
+  font-size: 16px;
+}
+
+.joinus-text{
+  font-family: "Poppins", "Roboto", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 26px;
+  color: #fff;
+}
+
+.logo-footer-text{
+  font-family: "Poppins", "Roboto", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 30px;
+  line-height: 42px;
+}
+
+.other-nav, .documentation-heading{
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.other-nav-list, .documentation-list, .footer-copy{
```

Comment:
Avoid selectors with more than two classes - ideally the selector should consist of a single class